examples/nanocoap_server/coap_handler.c

```diff
@@ -99,9 +99,15 @@ static ssize_t _riot_value_handler(coap_pkt_t *pkt, uint8_t *buf, size_t len, vo
     {
         /* convert the payload to an integer and update the internal value */
         char payload[16] = { 0 };
-        memcpy(payload, (char*)pkt->payload, pkt->payload_len);
-        internal_value = strtol(payload, NULL, 10);
-        code = COAP_CODE_CHANGED;
+        if (pkt->payload_len < 16) {
```

Comment:
You can now simplify the code a bit and move `char payload[16] = { 0 };` inside this `if` block

Reply:
Ok. I moved this line into if block.

Mayby a stupid question, but how can I now change commit message? Or I should use this one suggested by you in new commit with described here changes?

Reply:
You can use `git commit --amend` to edit the commit message of the last commit or apply staged changes to the last commit. (That is, changes you selected in `git gui` or changes in files you added with `git add`)

If you have several commits, you can use `git rebase -i `  and then the commit you want to rebase the branch onto.
E.g. `git rebase -i HEAD~3` if you want to edit the 3 last commits or `git rebase -i master` if you want to rebase onto the master branch.

This will open an editor where you can select what action to take with the individual commits.
You can also squash multiple commits into one.
There is even a shortcut for this: You can create fixup commits with `git commit --fixup=<ID of commit you want to fix>` those can then be automatically squashed with e.g `git rebase -i --autosquash HEAD~3`